src/ansiblelint/rules/FQCNBuiltinsRule.py

```diff
@@ -0,0 +1,131 @@
+"""Rule definition for usage of fully qualified collection names for builtins."""
+import sys
+from typing import Any, Dict, Optional, Union
+
+from ansiblelint.file_utils import Lintable
+from ansiblelint.rules import AnsibleLintRule
+from ansiblelint.testing import RunFromText
+
+builtins = [
+    "add_host",
+    "apt",
+    "apt_key",
+    "apt_repository",
+    "assemble",
+    "assert",
+    "async_status",
+    "blockinfile",
+    "command",
+    "copy",
+    "cron",
+    "debconf",
+    "debug",
+    "dnf",
+    "dpkg_selections",
+    "expect",
+    "fail",
+    "fetch",
+    "file",
+    "find",
+    "gather_facts",
+    "get_url",
+    "getent",
+    "git",
+    "group",
+    "group_by",
+    "hostname",
+    "import_playbook",
+    "import_role",
+    "import_tasks",
+    "include",
+    "include_role",
+    "include_tasks",
+    "include_vars",
+    "iptables",
+    "known_hosts",
+    "lineinfile",
+    "meta",
+    "package",
+    "package_facts",
+    "pause",
+    "ping",
+    "pip",
+    "raw",
+    "reboot",
+    "replace",
+    "rpm_key",
+    "script",
+    "service",
+    "service_facts",
+    "set_fact",
+    "set_stats",
+    "setup",
+    "shell",
+    "slurp",
+    "stat",
+    "subversion",
+    "systemd",
+    "sysvinit",
+    "tempfile",
+    "template",
+    "unarchive",
+    "uri",
+    "user",
+    "wait_for",
+    "wait_for_connection",
+    "yum",
+    "yum_repository",
+]
+
+
+class FQCNBuiltinsRule(AnsibleLintRule):
+    id = "fqcn-builtins"
+    shortdesc = "Use fqcn for builtins"
```

Comment:
```suggestion
    shortdesc = "Use FQCN for builtin actions"
```
The rule does not check plugins.